msupply-foundation/open-msupply
# 3043 generate outbound return lines

Fixes #3043

# 👩🏻‍💻 What does this PR do? 
 

Implements API for generateOutboundReturnLines - so we see real lines in the UI now 🎉 

![Screenshot 2024-02-23 at 1 16 40 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/1a826e28-482f-4319-a573-169dbb8d5cb2)

Also implements `item_id` filter for when implement the add from item view, but haven't put the return_id piece in yet, as we have no returns to query 😁 

# 🧪 How has/should this change been tested? 


Unit tests added for the service layer... should I be adding from graphql layer too?

- Go to a delivered/finalised inbound shipment
- Select some lines
- Choose `return selected lines`
- See those lines populated in the returns modal

## 💌 Any notes for the reviewer?
